### PR TITLE
Home

### PR DESCRIPTION
**Pull request by Hacene010**

### CODE REVIEW

**Review by christopheluc62 — DISMISSED** [review]

File: src/components/scss/Description.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.bloc-description{
+  display: flex;
+  flex-direction: row-reverse; /*Pour placer la description sur la gauche*/
```
Comment (LoicBrassart) [inline src/components/scss/Description.scss]: Comments in English plz

File: src/components/scss/Description.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    color: #fff;
+    width: 60vw;
+    height: 35vh;
+    margin-top:25vh;
+    margin-bottom: 25vh;
+    padding:20px;
+    text-align: justify;
+    box-shadow: 10px 10px 5px rgb(95, 92, 92);
+    
+
```
Comment (LoicBrassart) [inline src/components/scss/Description.scss]: Useless empty lines

File: src/components/scss/Description.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    padding:20px;
+    text-align: justify;
+    box-shadow: 10px 10px 5px rgb(95, 92, 92);
+    
+    
+  }
+  @media (max-width:700px) {
+    .Description{
+    font-size: x-small;
+    padding-top: 0;
```
Comment (LoicBrassart) [inline src/components/scss/Description.scss]: Indentation

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor Evils found ! 
Please fix it :smiling_imp:

File: src/components/scss/Description.scss (hunk 1 above)
Comment (Hacene010) [reply src/components/scss/Description.scss]: fixed it

File: src/components/scss/Description.scss (hunk 3 above)
Comment (Hacene010) [reply src/components/scss/Description.scss]: all fixed

**Review by christopheluc62 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Much better, No More Evil ;-)

**Pull request merged**